Example/FLEXample/Supporting Files/Info.plist

```diff
@@ -41,7 +41,7 @@
 					<key>UISceneConfigurationName</key>
 					<string>Default Configuration</string>
 					<key>UISceneDelegateClassName</key>
-					<string>$(PRODUCT_MODULE_NAME).SceneDelegate</string>
+					<string>$(PRODUCT_NAME).SceneDelegate</string>
```

Comment:
Can you leave a comment in this thread as to why this change was required?

Reply:
Bazel doesn't support this variable by default because, for targets like AppExtensions, We create 2 targets in Bazel and that creates 2 module names.
- `ios_extension` 
- and the underlying `swift_library`, so it can not figure out which module name to use hence they recommend explicitly setting value for this in Plist instead of using this variable , more info - https://github.com/bazelbuild/rules_apple/blob/master/doc/common_info.md#variable-substitution

Reply:
Since most of the cases PRODUCT_MODULE_NAME & PRODUCT_NAME would be same unless we explicitly override module name, hence I am using PRODUCT_NAME variable here.